cmd/lifecycle/analyzer.go

```diff
@@ -107,6 +108,17 @@ func (a *analyzeCmd) Exec() error {
 	return nil
 }
 
+func (a *analyzeCmd) setEnvRegistryAuth() {
```

Comment:
The addition of this function is covered in the analyzer acceptance tests. All other phases are untested...